spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/ConfigCheck.scala

```diff
@@ -56,9 +64,10 @@ object ConfigCheck {
    *
    * Fuzziness is determined by MatchThreshold
    */
-  def getSuggestedVars(unknownProp: String): Option[Seq[String]] = {
+  def getSuggestedVars(unknownProp: String, extraProps: Set[String] = Set.empty): Option[Seq[String]] = {
```

Comment:
I know you didn't write this, i'm just seeing it for the first time.
It is odd to return an Option of some collection when it should simply be Seq.empty in this case